python/paddle/fluid/tests/book/high-level-api/understand_sentiment/notest_understand_sentiment_stacked_lstm.py

```diff
@@ -69,48 +70,57 @@ def train_network(word_dict):
     return avg_cost, accuracy
```

Comment:
should be `[avg_cost, accuracy]` like [here](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/book/high-level-api/recognize_digits/test_recognize_digits_mlp.py#L44)

Reply:
Yes, thanks.